native/src/hooks/useTtsPlayer.ts

```diff
@@ -9,33 +9,22 @@ import { AppContext } from '../contexts/AppContextProvider'
 
 const useTtsPlayer = (model?: PageModel | LocalNewsModel | TunewsModel | undefined): TtsContextType => {
   const { languageCode } = useContext(AppContext)
-  const { setSentences, visible, setVisible, enabled, canRead } = useContext(TtsContext)
-  const sentences = useMemo(() => {
-    if (model) {
-      const content = parseHTML(model.content)
-      return [model.title, ...segment(languageCode, content)]
-    }
-
-    return []
-  }, [model, languageCode])
+  const ttsContext = useContext(TtsContext)
+  const [previousSentences] = useState(ttsContext.sentences)
```

Comment:
I did not find another way to ensure that the correct sentences are set after navigating back.